tests/integration-tests/src/test/resources/features/credential/sdjwt/present_proof.feature

```diff
@@ -0,0 +1,35 @@
+@sdjwt @proof
+Feature: Present SD-JWT Proof Protocol
+
+  Scenario Outline: Holder presents sd-jwt proof to <verifier>
+    Given <verifier> and Holder have an existing connection
+    And Holder has a sd-jwt issued credential from Issuer
+    When <verifier> sends a request for sd-jwt proof presentation to Holder requesting [firstName] claims
+    And Holder receives the presentation proof request
+    And Holder makes the sd-jwt presentation of the proof disclosing [firstName] claims
+    Then <verifier> has the proof verified
+    Examples:
+      | verifier |
+      | Verifier |
```

Comment:
What is the purpose of two verifiers: `verifier` and `Verifier`?

Reply:
the `verifier` is the title of the examples, `Verifier` is the actual actor